examples/eap74/todo-backend/todo-backend-s2i.yaml

```diff
@@ -0,0 +1,80 @@
+# This configuration file can be used to build and deploy the todo-backend
+# quickstart on OpenShift with the Helm Chart for EAP.
+build:
+  uri: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+  ref: EAP_7.4.0.GA
```

Comment:
> Cloning "https://github.com/jboss-developer/jboss-eap-quickstarts.git" ...
error: fatal: couldn't find remote ref EAP_7.4.0.GA

I suppose this should be updated as well as it was made in `helloworld-rs-app.yaml` file

Reply:
fixed, thanks